.dev_scripts/test_init_backbone.py

```diff
@@ -0,0 +1,280 @@
+        for i in range(len(names)):
+            if 'state_dict' in checkpoint:
+                state_dict = checkpoint['state_dict']
+            else:
+                state_dict = checkpoint
+            weight_sum = state_dict[checkpoint_names[i]].sum()
+            if layer_indexes[i] is None:
+                if layers[i] is None:
+                    after_init_weight_sum = getattr(model.backbone,
+                                                    names[i]).weight.sum()
+                else:
+                    layer = getattr(model.backbone, layers[i])
+                    after_init_weight_sum = getattr(layer,
+                                                    names[i]).weight.sum()
+            else:
+                if darknet_indexs[i] is not None:
+                    layer = getattr(model.backbone, layers[i])
+                    conv = getattr(layer, darknet_indexs[i])
+                    conv = getattr(conv, layer_indexes[i])
+                    after_init_weight_sum = getattr(conv,
+                                                    names[i]).weight.sum()
+                else:
+                    if names[i] is None:
+                        conv = getattr(model.backbone,
+                                       layers[i])[int(layer_indexes[i])]
+                        after_init_weight_sum = conv.weight.sum()
+                    else:
+                        layer = getattr(model.backbone,
+                                        layers[i])[int(layer_indexes[i])]
+                        after_init_weight_sum = getattr(layer,
+                                                        names[i]).weight.sum()
+            assert weight_sum == after_init_weight_sum
+        if print_cfg:
+            print('-' * 10 + 'Successfully load checkpoint' + '-' * 10 +
+                  '\n', )
+            return None
+    else:
+        if print_cfg:
+            print(config + '\n' + '-' * 10 +
+                  'config file do not have init_cfg' + '-' * 10 + '\n')
+            return config
+
+
+check_cfg_names = _traversed_config_file()
+
+
+@pytest.mark.parametrize('config', check_cfg_names)
+def test_load_pretrained(config):
+    """Check out backbone whether successfully load pretrained model by using
+    `backbone.init_cfg`.
+
+    Details please refer to `_check_backbone`
+    """
+    _check_backbone(config, print_cfg=False)
+
+
+def _test_load_pretrained():
+    """We traversed all potential config files under the `config` file. If you
+    need to print details or debug code, you can use this function.
+
+    Returns
+        check_cfg_names (list[str]): Config files that backbone initialized
+            from pretrained checkpoint might be problematic. Need to recheck
+            the config file. The output including the config files that the
+            backbone.init_cfg is None
+    # >>> from test_init_backbone import _test_load_pretrained
```

Comment:
Please follow the format of  [`Example`](https://github.com/open-mmlab/mmdetection/blob/5f6134796ed07f443d1bf389f57d3cee40bbd044/mmdet/core/anchor/anchor_generator.py#L41) or just remove it